l2os/cmd/l2os/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimistic-specs/l2os"
+	"github.com/ethereum-optimism/optimistic-specs/l2os/flags"
+)
+
+var (
+	Version   = ""
+	GitCommit = ""
+	GitDate   = ""
+)
+
+func main() {
+	// Set up logger with a default INFO level in case we fail to parse flags,
+	// otherwise the final critical log won't show what the parsing error was.
+	log.Root().SetHandler(
+		log.LvlFilterHandler(
+			log.LvlInfo,
+			log.StreamHandler(os.Stdout, log.TerminalFormat(true)),
+		),
+	)
+
+	app := cli.NewApp()
+	app.Flags = flags.Flags
+	app.Version = fmt.Sprintf("%s-%s", Version, params.VersionWithCommit(GitCommit, GitDate))
```

Comment:
FYI this pulls from geth's hardcoded version.

Reply:
yep

Reply:
Fixed, just used a simple `<version>-<commit>-<date>` format